templates/import_form.twig

```diff
@@ -69,6 +69,33 @@ jQuery(document).ready(function($) {
         }
     }
 
+    // Enables the confirm dialog when labels have been modified 
+    $('#labels-form').on('change', function(){
+        window.onbeforeunload = confirmOnPageExit;
+    });
+
+    // Disables the confirm dialog when saving the changes
+    $('#save-button, #save-button-bottom').click(function(){
+        window.onbeforeunload = null;
+    }); 
+
+    var confirmOnPageExit = function (e) 
+    {
+    // If we haven't been passed the event get the window.event
+    e = e || window.event;
+
+    var message = 'Any text will block the navigation and display a prompt';
+
+    // For IE6-8 and Firefox prior to version 4
```

Comment:
Ah ya!!  Stupid of me, I misunderstood. I got confused with the green highlighted lines. I thought that was made by you when you sent the message but I see those were my changes 😂 . Sorry, I'm not very familiar with Issues and Pull Requests Github interface yet.